tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.csproj

```diff
@@ -19,9 +23,15 @@
   </ItemGroup>
 
   <ItemGroup>
+    <EmbeddedResource Include="public_privatekey.pfx" />
```

Comment:
Not sure how people feel about using snake case to name files. Do you reckon it would be a good idea to stick to a different casing that would be consistent with other files? 

public_privatekey.pfx -> PublicPrivateKey.pfx